src/components/step-sequence/step-sequence.spec.tsx

```diff
@@ -1,11 +1,50 @@
 import React from "react";
+import { mount } from "enzyme";
 import TestRenderer from "react-test-renderer";
+import Logger from "../../__internal__/utils/logger";
 
 import StepSequence, { StepSequenceProps } from "./step-sequence.component";
 import StepSequenceItem from "./step-sequence-item/step-sequence-item.component";
 import { testStyledSystemSpacing } from "../../__spec_helper__/test-utils";
 
 describe("StepSequence", () => {
+  describe("Deprecation warning", () => {
```

Comment:
comment(non-blocking): I don't think the Deprecation Warning describe block really offers much extra and could probably be removed